docs/docs/dev/networking/cni.md

```diff
@@ -1,44 +1,270 @@
-  --set cni.enabled=true,cni.chained=true,cni.netDir="/etc/cni/net.d",cni.binDir=/opt/cni/bin,cni.confName=10-calico.conflist \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-kindnet.conflist \
+   kuma kuma/kuma
+```
+
+:::
+::::
+:::::
+
+::::: tab "Azure"
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
+
+```shell
+kumactl install control-plane \
+  --set "cni.enabled=true" \
+  --set "cni.chained=true" \
+  --set "cni.netDir=/etc/cni/net.d" \
+  --set "cni.binDir=/opt/cni/bin" \
+  --set "cni.confName=10-azure.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-azure.conflist \
+   kuma kuma/kuma
+```
+
+:::
+::::
+:::::
+
+::::: tab "AWS - EKS"
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
+
+```shell
+kumactl install control-plane \
+  --set "cni.enabled=true" \
+  --set "cni.chained=true" \
+  --set "cni.netDir=/etc/cni/net.d" \
+  --set "cni.binDir=/opt/cni/bin" \
+  --set "cni.confName=10-aws.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-aws.conflist \
    kuma kuma/kuma
 ```
 
 :::
 ::::
+:::::
+
+::::: tab "Google - GKE"
+
+You need to [enable network-policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy) in your cluster (for existing clusters this redeploys the nodes).
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
+
+```shell
+kumactl install control-plane \
+  --set "cni.enabled=true" \
+  --set "cni.chained=true" \
+  --set "cni.netDir=/etc/cni/net.d" \
+  --set "cni.binDir=/home/kubernetes/bin" \
+  --set "cni.confName=10-calico.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/home/kubernetes/bin,cni.confName=10-calico.conflist \
+   kuma kuma/kuma
+```
+
+:::
+::::
+:::::
+
+::::: tab "OpenShift 3.11"
+
+1. Follow the instructions in [OpenShift 3.11 installation](../installation/openshift/#_2-run-kuma)
+to get the `MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook` enabled (this is required for regular kuma installation).
+
+2. You need to grant privileged permission to kuma-cni service account:
+
+```shell
+oc adm policy add-scc-to-user privileged -z kuma-cni -n kube-system
+```
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
+
+```shell
+kumactl install control-plane \
+  --set "cni.enabled=true" \
+  --set "cni.containerSecurityContext.privileged=true"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.securityContext.privileged=true \
+   kuma kuma/kuma
```

Comment:
```suggestion
helm install 
  --namespace kuma-system \
  --set "cni.enabled=true" \
  --set "cni.containerSecurityContext.privileged=true" \
    kuma kuma/kuma
```

Reply:
In this suggestion I also changed `cni.securityContext` to `cni.containerSecurityContext` which is correct one